astropy/modeling/tests/test_parameters.py

```diff
@@ -127,226 +125,255 @@ def test_parameter_operators():
     assert -par == -num
     assert abs(par) == abs(num)
 
-
```

Comment:
Why is the following section removed from the tests? It shows a change in functionality (see below).

Reply:
Put back in.